addons/fleet/views/fleet_vehicle_views.xml

```diff
@@ -256,67 +256,43 @@
-                                </div>
-                                <ul>
-                                    <li>
-                                        <t t-if="record.driver_id.raw_value"><field name="driver_id" widget="many2one_avatar" options="{'display_avatar_name': True}"/></t>
-                                    </li>
-                                    <li>
-                                        <t t-if="record.future_driver_id.raw_value">Future Driver : <field name="future_driver_id"/></t>
-                                    </li>
-                                    <li>
-                                        <t t-if="record.location.raw_value"><small><i class="fa fa-map-marker" title="Location"></i> <field name="location"/></small></t>
-                                    </li>
-                                    <li>
-                                        <field name="vehicle_properties" widget="properties"/>
-                                    </li>
-                                </ul>
-                                <div class="o_kanban_record_bottom" t-if="!selection_mode">
-                                    <div class="oe_kanban_bottom_left">
-                                        <a t-if="record.contract_count.raw_value>0" data-type="object"
-                                           data-name="return_action_to_open" href="#" class="oe_kanban_action oe_kanban_action_a"
-                                           data-context='{"xml_id":"fleet_vehicle_log_contract_action"}'>
-                                            <field name="contract_count"/>
-                                            Contract(s)
-                                            <span t-if="record.contract_renewal_due_soon.raw_value and !record.contract_renewal_overdue.raw_value"
-                                                class="fa fa-exclamation-triangle text-warning" role="img" aria-label="Warning: renewal due soon" title="Warning: renewal due soon">
-                                            </span>
-                                             <span t-if="record.contract_renewal_overdue.raw_value"
-                                                class="fa fa-exclamation-triangle text-danger" role="img" aria-label="Attention: renewal overdue" title="Attention: renewal overdue">
-                                            </span>
-                                        </a>
-                                        <field name="activity_ids" widget="kanban_activity"/>
-                                    </div>
-                                </div>
+                            <field name="tag_ids" widget="many2many_tags" options="{'color_field': 'color'}"/>
+                            <field t-if="record.driver_id.raw_value" name="driver_id" widget="many2one_avatar" options="{'display_avatar_name': True}" class="small"/>
```

Comment:
t-if not necessary I think

Reply:
if we remove the t-if -> if there is no driver then assign driver symbol will be visible see :
![image](https://github.com/odoo/odoo/assets/120459808/1aac51d0-10a0-497a-ac47-2d93ecd60b8a)
with the t-if -> there is no driver then also symbol is not visible

Reply:
ok